data_models/summarizer_output.py

```diff
@@ -92,3 +91,16 @@ def get_formatted_dict(self):
     def get_weighted_text_index(self):
         return self.para_index \
             + self.sentence_in_para_index * self.sentence_in_para_weight
+
+
```

Comment:
I see that `get_weighted_text_index` is same as in `data_models/sentence_with_attributes.py`
Can you avoid duplicate code?

Reply:
I think you should pass `SentenceWithAttributes` object to `StampPage` constructor.

Reply:
that's a good suggestion but doing so would mean we've to possibly change summarizer output fields which would mean the generator would also have to be tweaked. WDYT ?

Reply:
Yes, change that as well.

Reply:
As per discussion with @KomalBhadani27 , will amend this in a separate PR . this will require some considerable changes to generator and needs further discussion. hope this is okay!